docs/driver.md

```diff
@@ -33,9 +33,9 @@ Empty by default. Used by the driver to further customize the IDE.
 #### Screen size
 ```
 driver {
-  screen {
-    width = 1920,
-    height = 1080,
+  xvfb-screen {
```

Comment:
I'd say that Xvfb mode itself should be configurable via HOCON, and not just IDEPROBE_DISPLAY env var, for the sake of consistency &mdash; pls open an issue for that @azdrojowa123.

To avoid having to make a breaking change when implementing that, I'd now do sth like:
```
display {
  xvfb {
    // enabled = true -> to be added one day, not now
   screen {
      width = 1920
      height = 1080
      depth = 24
    }
  }
}
```

@lukaszwawrzyk WDYT?

Reply:
yes, and we can have a default that reads env var.